metta/rl/trainer.py

```diff
@@ -326,10 +327,36 @@ def train(self) -> None:
         trainer_cfg = self.trainer_cfg
 
         if self._stats_client is not None:
-            name = self.wandb_run.name if self.wandb_run is not None and self.wandb_run.name is not None else "unknown"
+            # Generate a unique name when WandB fails to avoid silent training run creation failures
```

Comment:
Extract name generation to a separate PR, this is unrelated to the feature at hand